spec/controllers/ops_controller/settings/common_spec.rb

```diff
@@ -231,6 +231,19 @@
           expect(edit[:new]).to eq(edit[:current])
         end
       end
+
+      context 'zone node' do
+        it 'sets ntp server info for display' do
+          controller.instance_variable_set(:@sb, :active_tab => 'settings_zone')
+          controller.instance_variable_set(:@edit, :new => {:ntp => {:server => ["zone1.pool.com", "zone2.pool.com"]}})
+          @zone = FactoryGirl.create(:zone)
+          controller.send(:zone_save_ntp_server_settings, @zone)
+          @miq_server = FactoryGirl.create(:miq_server, :zone => @zone)
+          allow(MiqServer).to receive(:my_server).and_return(@miq_server)
```

Comment:
We already have a helper to creates everything you need here.  `_guid, miq_server, zone = EvmSpecHelper.local_guid_miq_server_zone`